src/Symfony/Component/DependencyInjection/Tests/Fixtures/includes/autowiring_classes.php

```diff
@@ -352,3 +354,32 @@ public function setDefaultLocale($defaultLocale)
     {
     }
 }
+
+interface DecoratorInterface
+{
+}
+
+class Decorated implements DecoratorInterface
+{
+    public function __construct($quz = null, \NonExistent $nonExistent = null, DecoratorInterface $decorated = null, array $foo = array())
+    {
+    }
+
+    public static function create(\NonExistent $nonExistent = null, $factory = null)
```

Comment:
what's that used for?

Reply:
Indeed it's useless now (it was a copy paste of the test of the previous implementation)